src/plugins/discover/public/application/components/utils/local_storage.ts

```diff
@@ -5,7 +5,7 @@
 
 import { Storage } from '../../../../../opensearch_dashboards_utils/public';
 
-export const NEW_DISCOVER_KEY = 'discover:newExpereince';
+export const NEW_DISCOVER_KEY = 'discover:newExperience';
```

Comment:
Yeah this always stuck out like a red thumb for me. One risk to call out here is that previous users who set this will need to reset this. Should we call this out in a changelog?

Reply:
Something like that?

> fixed typo in option name that disabled legacy Discover UI. Those using new Discover have to re-enable it once again

Reply:
yep that works